Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1772,6 +1774,8 @@ def replace_str(src_str):
            r'(?:(?:\/|\?)[-\w\d+&@#\/%=~_$?!\-:,.\(\);]*[\w\d+&@#\/%=~_$\(\);])?'
 cveRegex = r'(?i)^cve-\d{4}-([1-9]\d{4,}|\d{4})$'
 
+domainGlobRegEx = r'(\*\.)([\w\d]+\.)+[\w\d]+\b'
```

Comment:
Not sure what code is supposed to use this regex. But I don't think we need a regex here. The logic is very simple. If a domain contains the `*` character anywhere in the string then we should treat it as a DomainGlob and not as a regular Domain.

Reply:
From what I have read the domain `abc*def.google.com` is not a glob domain. However, if this falls under our definition of domainGlob then we can do that.
@avidan-H you thoughts